packages/xdl/src/Web.js

```diff
@@ -70,6 +57,61 @@ export async function hasWebSupportAsync(projectRoot) {
   return isWebConfigured;
 }
 
+async function getMissingReactNativeWebPackagesMessageAsync(projectRoot) {
+  const { pkg } = await readConfigJsonAsync(projectRoot);
+  const dependencies = await getMissingReactNativeWebPackagesAsync(projectRoot, pkg);
+  if (dependencies.length) {
+    const commandPrefix = ConfigUtils.isUsingYarn(projectRoot) ? 'yarn add ' : 'npm install ';
+    const command = commandPrefix + dependencies.join(' ');
+    return {
+      message: `* Install these packages: ${chalk.yellow(command)}`,
+      command,
+      dependencies,
+    };
+  }
+}
+
+function formatPackage({ name, version }) {
+  if (!version || version === '*') {
+    return name;
+  }
+  return `${name}@${version}`;
+}
+
+async function getMissingReactNativeWebPackagesAsync(projectRoot, appPackage) {
+  const { dependencies = {} } = appPackage;
+  const requiredPackages = [
```

Comment:
Talking to @fson, it might make sense to add react-dom and react-native-web versions to our `/versions` endpoint, similar to what we do for specifying the correct version of RN for iOS/Android.

Reply:
how would I do this?

Reply:
Probably need to look in www to see how we update the versions endpoint and how it is served so that when we update it for SDK 33, info about the the web dependencies gets sent back as well.

For the client side, I think XDL already needs to know about the versions endpoint when creating a new project -> grep around to see what functions we already have for working with the endpoint.